resources/puppetlabs/lein-ezbake/template/global/install.sh.erb

```diff
@@ -27,6 +27,7 @@ symbindir=${symbindir:=/opt/puppetlabs/server/bin}
 app_prefix=${app_prefix:=/opt/puppetlabs/server/apps/${real_name}}
 app_data=${app_data:=/opt/puppetlabs/server/data/${real_name}}
 app_logdir=${app_logdir:=/var/log/puppetlabs/${real_name}}
+system_config_dir=${system_config_dir:=${app_prefix}/config}
```

Comment:
@stahnma so you are good with, e.g., `/opt/puppetlabs/server/apps/puppetserver/config` for the "hidden" config files?  Just wanted to make absolutely sure before we merged it.